app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,69 @@
+class RecipeFoodsController < ApplicationController
+  before_action :set_recipe_food, only: %i[show edit update destroy]
+
+  # GET /recipe_foods or /recipe_foods.json
+  def index
+    @recipe_foods = RecipeFood.all
+  end
+
+  # GET /recipe_foods/1 or /recipe_foods/1.json
+  def show; end
+
+  # GET /recipe_foods/new
+  def new
+    @recipe_food = RecipeFood.new
+  end
+
+  # GET /recipe_foods/1/edit
+  def edit; end
+
+  # POST /recipe_foods or /recipe_foods.json
+  def create
+    @recipe_food = RecipeFood.new(recipe_food_params)
+
+    respond_to do |format|
+      if @recipe_food.save
+        format.html { redirect_to recipe_food_url(@recipe_food), notice: 'Recipe food was successfully created.' }
+        format.json { render :show, status: :created, location: @recipe_food }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @recipe_food.errors, status: :unprocessable_entity }
+      end
+    end
```

Comment:
![image](https://github.com/zbahati/recipeApp/assets/93039359/dcadc9d1-2627-4fed-8892-02d0bd22cd02)

- [ ] Great work so far but I faced an issue while creating the food in a recipe :fearful: When I try to access the recipe_food form using the Add Ingredient button I see an error which you can see in the image above :eyes: Kindly check this and fix it to make your project work as expected :+1:

Reply:
I have fixed this errors was the typo error for the current_user.food where should be current_user.foods